pages/about.html

```diff
@@ -5,8 +5,30 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/styles.css" rel="stylesheet">
 </head>
 <body>
-  
+  <header>
+    <h1>Justina Liu</h1>
+    <nav>
+            <div class="nav-flex-link" >
```

Comment:
The `div`s wrapping the anchors don't seem to be necessary. They're hugging the `a` tags as-is. If they're removed, the `a` tags themselves become the children of teh `flex` and get your spaced layout.